mage_ai/io/base.py

```diff
@@ -51,6 +51,7 @@ class ExportWritePolicy(str, Enum):
     APPEND = 'append'
     FAIL = 'fail'
     REPLACE = 'replace'
+    UPSERT = 'upsert'
```

Comment:
do we need another policy?
the merge command is applied in postgres and mssql when policy='append' and unique constraints are specified.

Reply:
ok i can do that for consistency